### ISSUE TITLE
Can't filter raw IMS data without losing ion mobility dimension

### ISSUE THREAD

**Issue by kholmes624** [mzmine/mzmine2#977]
* My operating system: Windows 10 Pro
* My MZmine version: 3.4.27

I have really large TIMS-TOF files, and I'm trying to filter the raw data to reduce processing times. But I can't find a way to do so with the existing raw data filter modules. 

If I use "crop filter" and set the "Base filtering integer" to 10, then I get an output file. But if I ask for the ion mobility data overview, I see this:
![image](https://user-images.githubusercontent.com/38500909/234921525-87a33ab4-8885-4999-910e-e1901211cb4b.png)
Implying that in order to filter out 1 in every 10 scans, MZmine completely collapses the ion mobility dimension of the data.

If instead I use scan-by-scan filtering, I am also unable to see the IMS dimension, and get an additional warning:
![image](https://user-images.githubusercontent.com/38500909/234924077-de2d2d24-e04c-4f2d-a7d1-8690e0bd2676.png)

This one I find hard to understand, because the raw files are .tdf and I did not use MSConvert.

Long story short: Is there a way to subsample/filter IMS scans in MZMine, or is it something still in the pipeline? Are there any other ways to go about data reduction before you start a full workflow?

**Comment by SteffenHeu** [comment]
Hi kholmes,

both modules should give you a warning that they were not tested with IMS data. (Or have given it on first start) They were not designed for IMS data and have not been updated yet.

You could try the "base filtering integer" in the scan filters when running the ADAP chromatogram builder. It will only use a scan if the scan number is divisible by the "base filtering integer". This can lead to irregular RT intervals though, if you have varying number of MS2 frames.

We have not had a problem with too large timsTOF files yet for LC-IMS-MS experiments. Do you know what the reason is your files are so large?

**Comment by kholmes624** [comment]
Hi,

Thank you for your quick reply! I saw the warnings that the modules had not been tested with IMS data, but couldn't figure out any alternative ways to filter the data. I will try using the filter at the chromatogram builder stage, thanks! If I have data-independent acquisition of the MS2 data, should that resolve the RT interval issue?

MZMine works well with just a single file, but slows considerable with multiple. I also just found out that these particular samples were run using LC-TIMS-DIA-PASEF-MS/MS. From my brief reading it seems the PASEF method makes the files huge? I can't find any documentation on PASEF in MZMine. Would it be best to use a different tool?

**Comment by SteffenHeu** [comment]
Hi,

regular PASEF works, but MZmine is not optimized for dia PASEF yet. I'd recommend MS-Dial for that or to acquire the files using regular (dda) PASEF. Then you can use the workflow wizard and start from there.

**Comment by kholmes624** [comment]
Thank you for the recommendation!
